Lib/fontTools/varLib/mutator.py

```diff
@@ -48,8 +48,8 @@ def main(args=None):
 			coordinates += GlyphCoordinates(var.coordinates) * scalar
 		_SetCoordinates(varfont, glyphname, coordinates)
 
-	print("Removing GX tables")
-	for tag in ('fvar','avar','gvar'):
+	print("Removing variable tables")
+	for tag in ('fvar','avar','gvar','HVAR'):
```

Comment:
I meant, you can go ahead and drop VVAR, STAT, MVAR, etc as well. And yes, mutator is very much a WIP. But then again, so is the rest of varLib. But mutator is the least complete one.